app/src/main/java/com/mstudio/superheroarch/MainActivity.kt

```diff
@@ -34,30 +44,76 @@ class MainActivity : AppCompatActivity() {
         }
 
         val apiRick = retrofit.create(ApiRick::class.java)
-        val titleTextView = findViewById<TextView>(R.id.title)
+        val name = findViewById<TextView>(R.id.character_name)
+        val status = findViewById<TextView>(R.id.character_status)
         val button = findViewById<Button>(R.id.main_button)
         val context = this
+        val displayMetrics = resources.displayMetrics
+        val screenWidth = displayMetrics.widthPixels
+        val imageView = findViewById<ImageView>(R.id.character_image)
+        val desiredSize = (screenWidth * 0.2).toInt()
```

Comment:
This is really overkill. 

To achieve this you need to change the XML. Put 0dp as width and height, then constraint_layout_percentaje (or something similar) to 0.2 and finally there is another property called aspect_ratio "1:1". Furthermore, you'll need to play with the scale properties.

Reply:
Ah, you already have it, then why you have this?